src/LibYear/App.cs

```diff
@@ -8,7 +9,6 @@ public class App
 	private readonly IPackageVersionChecker _checker;
 	private readonly IProjectFileManager _projectFileManager;
 	private readonly IAnsiConsole _console;
-
```

Comment:
Let's get this blank line back

Reply:
Cleaning that up!
Apologies, I wonder if rider formatted on save or I ran dotnet format by habit or something, did not intend whitespace changes!